src/Session.h

```diff
@@ -421,8 +436,6 @@ class Session {
 
   void copy_state_to(Session& dest, EmuFs& emu_fs, EmuFs& dest_emu_fs);
 
-  // XXX Move CloneCompletion/CaptureState etc to ReplayTask/ReplaySession
```

Comment:
Do you mean that CloneCompletion/CaptureState should be nested types (as they were before) but inside ReplayTask/ReplaySession instead - or just moved to those headers as public types? (it used to be a nested type of `Session`)

I was thinking, if there is a possibility that this data can be re-used for persistent checkpoints made during record in the future (though, I'm just guessing here) I figured it be better to just have it as public types in `Session.h` instead of being nested and/or in just `ReplayTask/ReplaySession.h`. But maybe I should move it, and when it's time to cross that road, deal with it then instead?